src/pages/ExamsPage/components/ExamSelection.jsx

```diff
@@ -20,13 +19,32 @@ const ExamSelection = ({ exams, onSelect }) => {
         onChange={setSearchText}
         placeholder={formatMessage(messages.examSelectPlaceholder)}
         onSubmit={() => {}}
+        defaultSelected
```

Comment:
Note: so I tried to set a specific MenuItem as defaultSelected, which does seem to work as expected. But I cannot figure out how I would set defaultSelected to false (except for just not having this prop set for any MenuItem which is how it was before). I am guessing I would need to make the change here and should not be actually changing the SelectMenu paragon component?